src/js/store/projects/actions.ts

```diff
@@ -158,22 +165,63 @@ export const updateProject = (payload: Project): ProjectUpdated => ({
   payload,
 });
 
-export const projectError =
+export const projectCreated =
   ({
     model,
-    message,
     originating_user_id,
   }: {
     model: Project;
-    message?: string;
     originating_user_id: string | null;
   }): ThunkResult<ProjectUpdated> =>
   (dispatch, getState) => {
     if (isCurrentUser(originating_user_id, getState())) {
       dispatch(
         addToast({
           heading: t(
-            'Uh oh. There was an error re-syncing GitHub Collaborators for this Project: “{{project_name}}.”',
+            'Successfully created GitHub Repository for new Project: “{{project_name}}.”',
+            { project_name: model.name },
+          ),
+          linkText: model.repo_url ? t('View Project on GitHub.') : undefined,
+          linkUrl: model.repo_url ? model.repo_url : undefined,
+          openLinkInNewWindow: true,
+        }),
+      );
+    }
+
+    return dispatch(updateProject(model));
+  };
+
+export const projectCreateError =
+  ({
+    model,
+    message,
+    originating_user_id,
+  }: {
+    model: Project;
+    message?: string;
+    originating_user_id: string | null;
+  }): ThunkResult<ProjectDeleted> =>
+  (dispatch, getState, history) => {
+    const appState = getState();
+    const project = selectProjectById(appState, model.id);
+    const currentUser = isCurrentUser(originating_user_id, appState);
+
+    if (
+      history.location.pathname ===
+      routes.project_detail(project?.slug || model.slug)
```

Comment:
@jerivas For some reason the instance of the deleted project that's included in this WS event (`PROJECT_CREATE_ERROR`) has `slug: null`, so in this case to know the URL I'm finding the project in the local Redux store. I think it's an okay workaround, but I wanted to mention since it was unexpected.

Reply:
Sounds like we're hitting a race condition. The `PROJECT_CREATE_ERROR` event is emitted asynchronously and by the time it serializes the in-memory `Project` instance all slugs are gone from the DB and [`project.slug` falls back to `None`](https://github.com/SFDO-Tooling/sfdo-template-helpers/blob/main/sfdo_template_helpers/slugs.py#L59-L64).

Reply:
Ok -- we could dig into that, but it should work to rely on the front-end model to get the slug in this case. 👍